paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerAdapter.kt

```diff
@@ -82,7 +75,7 @@ interface CustomerAdapter {
      * have a default saved payment method.
      * @return the saved [PaymentOption].
      */
-    suspend fun retrieveSelectedPaymentOption(): Result<PaymentOption?>
+    suspend fun retrieveSelectedPaymentOption(isGooglePayAvailable: Boolean): Result<PaymentOption?>
```

Comment:
We can just pass this in directly from `CustomerSheet` rather than have a hacky approach to do it. I think merchants will also want to know if `Google Pay` is available on devices when fetching the selected payment option.